CHANGELOG_PENDING.md

```diff
@@ -2,6 +2,18 @@
 
 ## v0.34.21
 
+Release highlights include:
+- A new RPC configuration flag, `discard_abci_responses`, which, if enabled,
+  discards all ABCI responses except the latest one in order to reduce disk
+  space usage in the state store. Note that, if enabled, this means that the
+  `block_results` RPC endpoint will only be able to return the latest block.
+- A CLI command, `reindex-event`, is introduced to re-index block and tx events
+  to the event sinks. You can run this command when the event store backend
+  dropped/disconnected or you want to replace the backend. Note that, if
+  `discard_abci_responses` is enabled, you will not be able to use this command.
```

Comment:
I'd change the wording of the last sentence as above. 

I think "is introduced" is redundant for a changelog. In general change logs are concerned with additions to the product.